tests/SpecTests/CrudSpecTest.php

```diff
@@ -19,11 +19,6 @@ class CrudSpecTest extends FunctionalTestCase
         'aggregate-merge: Aggregate with $merge and majority readConcern' => 'PHPLIB-438',
         'aggregate-merge: Aggregate with $merge and local readConcern' => 'PHPLIB-438',
         'aggregate-merge: Aggregate with $merge and available readConcern' => 'PHPLIB-438',
-        'aggregate-out-readConcern: readConcern majority with out stage' => 'PHPLIB-431',
-        'aggregate-out-readConcern: readConcern local with out stage' => 'PHPLIB-431',
-        'aggregate-out-readConcern: readConcern available with out stage' => 'PHPLIB-431',
-        'aggregate-out-readConcern: readConcern linearizable with out stage' => 'PHPLIB-431',
-        'aggregate-out-readConcern: invalid readConcern with out stage' => 'PHPLIB-431',
```

Comment:
Related to APM discussions in #402, do you know if these failed after removing the skips? Since these tests _do_ include `expectations`.

Reply:
Yes, these tests failed as they expect the `readConcern` option to be present in the command.